generators/app/index.js

```diff
@@ -64,6 +64,12 @@ module.exports = generators.Base.extend({
       desc: 'Project name'
     });
 
+    this.option('githubAccount', {
```

Comment:
It's used in `node:readme`, but `node:app` asks for `githubAccount` whether or not it's provided